src/csharp/CSharpParser.g4

```diff
@@ -392,7 +394,7 @@ simple_embedded_statement
 
 	// selection statements
 	| IF OPEN_PARENS expression CLOSE_PARENS if_body (ELSE if_body)?               #ifStatement
-    | SWITCH OPEN_PARENS expression CLOSE_PARENS OPEN_BRACE switch_section* CLOSE_BRACE           #switchStatement
+  | SWITCH OPEN_PARENS expression CLOSE_PARENS OPEN_BRACE switch_section* CLOSE_BRACE           #switchStatement
```

Comment:
I'm not sure if indentation is incorrect or just GitHub reviews that make it look incorrect.